app/questionnaire/router.py

```diff
@@ -420,6 +423,7 @@ def get_next_block_url(
             block_id=next_block_id,
             list_name=routing_path.list_name,
             list_item_id=routing_path.list_item_id,
+            _external=False,
```

Comment:
Was this needed from some breaking change?. Is there a link to the doc?.

Reply:
Yes, I believe we discussed this in a tech session. 
[Flask Version 2.2.x](https://flask.palletsprojects.com/en/2.2.x/api/#flask.Flask.url_for)
[Flask Version 2.0.x](https://flask.palletsprojects.com/en/2.0.x/api/#:~:text=flask.-,url_for,-(endpoint%2C)
The parameters have been reorganised with Flask 2.2.x .
I had to specifically set `_external=False ` otherwise any vardic argument  would be passed as through the `_external` param rather then `**kwargs`.